ansible_runner/runner_config.py

```diff
@@ -288,10 +289,16 @@ def prepare_env(self):
             self.settings = dict()
 
         try:
-            self.ssh_key_data = self.loader.load_file('env/ssh_key', string_types)
+            self.ssh_key_data = [self.loader.load_file('env/ssh_key', string_types)]
         except ConfigurationError:
-            output.debug("Not loading ssh key")
-            self.ssh_key_data = None
+            try:
+                key_list = os.listdir(self.loader.abspath('env/ssh_key'))
```

Comment:
Instead of doing this we could do an `isdir(...)` to test if this is a directory and load the files up that way?

Reply:
Well, I am not a fan of nested try/excepts either. I could remove the second one and handle everything inside of that condition. However, having both cases handled with lists simplify the code a little.

Also, I am wondering is self.ssh_key_path is used somewhere else. In this new iteration, that value would not make much sense when having several keys. We could just use a dictionary with the path as the key.

Reply:
If you want to use sub folders in env/ssh_key, dump_artifacts(kwargs) in utils.py should be modified:

```suggestion
    for key in ('ssh_key', 'cmdline'):
        obj = kwargs.get(key)
            path = os.path.join(private_data_dir, 'env')
            if isinstance(obj, list):
                os.makedirs(os.path.join(path, key), mode=0o700)
                for idx, item in enumerate(obj):
                    dump_artifact(str(item), os.path.join(path, key), str(idx))
            else:
                dump_artifact(str(kwargs[key]), path, key)
            kwargs.pop(key)
```